lib/simd/Grid_generic_types.h

```diff
@@ -26,7 +26,7 @@ Author: Antonin Portelli <antonin.portelli@me.com>
     See the full license in the file "LICENSE" in the top level distribution directory
     *************************************************************************************/
     /*  END LEGAL */
-
+#define GEN_SIMD_WIDTH 16
```

Comment:
This can generate conflicts with QPX code 
@nmeyer-ur would you use the same definition style as in the QPX code and delete this define here?